crates/bevy_core_pipeline/src/bloom/mod.rs

```diff
@@ -130,36 +132,37 @@ impl Plugin for BloomPlugin {
 ///
 /// Often used in conjunction with `bevy_pbr::StandardMaterial::emissive`.
 ///
-/// Note: This light is not "real" in the way directional or point lights are.
+/// Should be used along with a tonemapping function that maps each RGB component separately, such as ACES Filmic.
+///
+/// ## Note
```

Comment:
I would remove this note entirely. Real bloom also happens in camera only and therefore wouldn't cast shadows or light other objects. This seems like a useful note to have for emissive materials (that might have been the original intention) so maybe it should be moved there.

Reply:
Heavily disagree on this. Most beginner gamedevs don't know what Bloom is - they look for "glow" or something, and eventually stumble upon bloom. I think it's very important to introduce the subject to people new to it, directly on the relevant camera setting, as with all rendering effects.

Reply:
In that case I would personally prefer a better description of what it does do instead of explaining what it doesn't do. I see how "Makes objects glow" could be misunderstood in that way. If you phrase it something like "Bloom emulates an effect  found in real cameras and the human eye that causes a halo around bright objects, making them appear even brighter" then I wouldn't expect anyone to think that bloom casts a shadow or lights up other objects.